frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -57,43 +57,33 @@ const defaultImplicitActionCreateOptions = {
 const enableImplicitActionsForModel =
   async (modelId: number, options = defaultImplicitActionCreateOptions) =>
   async (dispatch: Dispatch) => {
-    const requests = [];
-
     if (options.insert) {
-      requests.push(
-        ActionsApi.create({
-          name: t`Create`,
-          type: "implicit",
-          kind: "row/create",
-          model_id: modelId,
-        }),
-      );
+      await ActionsApi.create({
```

Comment:
Wait for each call sequentially, so the implicit actions are ordered deterministically.